SourceX/miniwin/misc_msg.cpp

```diff
@@ -5,6 +5,10 @@
 #include "stubs.h"
 #include <math.h>
 #include "../../touch/touch.h"
+#ifdef SWITCH
+	#include "ddraw.h"
```

Comment:
I this DirectDraw? Why would it be needed?

Reply:
It is the DD wrapper yes. I only needed `window` and `renderer` references from it. I fixed it just now, using externals instead.